src/_data/pricingFeatures.yaml

```
@@ -100,6 +112,12 @@ sections:
         selfHostedValues: [ 'check', 'check' ]
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p><a href='https://github.com/FlowFuse/nr-project-nodes' target='blank'>FlowFuse Project Nodes</a> enable the passing of data and messages between your Node-RED projects, included with all FlowFuse Projects by default.</p>"
+      - id: private-npm-registry
+        label: Private NPM Registry
+        cloudValues: [ null, 'check', 'check' ]
+        selfHostedValues: [ 'check', 'check' ]
+        tags: [ 'cloud', 'self-hosted' ]

```

@gstout52 I believe this is only available on FF Cloud https://flowfuse.com/changelog/2025/03/team-npm-registry/
```suggestion
        tags: [ 'cloud' ]
```

---

@Yndira-E We can merge as-is, as this will be available upon the next release.